com.unity.render-pipelines.high-definition/Editor/Compositor/CompositorWindow.cs

```diff
@@ -114,13 +114,13 @@ void OnGUI()
                 Undo.RegisterCreatedObjectUndo(compositor.outputCamera.gameObject, "Create Compositor");
                 Undo.RegisterCreatedObjectUndo(go, "Create Compositor");
             }
-            else if (compositor)
+            else if (compositor && (compositor.enabled != enableCompositor))
             {
                 string message = enableCompositor ? "Enable Compositor" : "Disable Compositor";
                 Undo.RecordObject(compositor, message);
                 compositor.enabled = enableCompositor;
```

Comment:
Note: This assignment was always creating an undo action, so we now do it only if necessary.